sys/androidmedia/gstamc-constants.h

```diff
@@ -101,7 +101,10 @@ enum
   /* From hardware/ti/omap4xxx/domx/omx_core/inc/OMX_TI_IVCommon.h */
   COLOR_TI_FormatYUV420PackedSemiPlanarInterlaced = 0x7f000001,
   COLOR_EXYNOS_FormatNV12Tiled = 0x7fc00002,
-  COLOR_EXYNOS_FormatNV21Linear = 0x7f000011
+  COLOR_EXYNOS_FormatNV21Linear = 0x7f000011,
+  /* MTK formats */
```

Comment:
These 2 formats have been added to OMX_COLOR_FORMATTYPE (OMX_IVCommon.h) by the SoC vendor. FormatYV12 is one of the color formats used by VP8/9 in arale. The other one is added for completeness. I'll add a comment in the code.